include/xmlwrapp/attributes.h

```diff
@@ -130,8 +130,13 @@ class XMLWRAPP_API attributes
     private:
         void *node_;
         void *prop_;
+
+        // Using std::string here is fine as long as both the main application and
+        // the DLL use the same CRT version, which should always be the case.
```

Comment:
Sorry, this is really trivial, but let's move a generic version of this comment to the declaration of the macro itself, to avoid repeating it too.

I.e. let's do this:
```suggestion
```